org.osgi.test.assertj.framework/src/main/java/org/osgi/test/assertj/versionrange/VersionBoundAssert.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.osgi.test.assertj.versionrange;
+
+import org.assertj.core.api.InstanceOfAssertFactory;
+import org.osgi.framework.Version;
+
+public class VersionBoundAssert extends AbstractVersionBoundAssert<VersionBoundAssert, Version> {
+	public static InstanceOfAssertFactory<Version, VersionBoundAssert>	OPEN_VERSION	= new InstanceOfAssertFactory<>(
```

Comment:
OPEN_VERSION and CLOSED_VERSION need to be final.

Reply:
Done!